circle.yml

```diff
@@ -1,6 +1,9 @@
 machine:
   ruby:
     version: 2.4.1
+  pre:
+    - sudo apt-get install libfontconfig
+    - sudo curl --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-2.1.1
```

Comment:
Yeah, this is an example of copy and paste from stackoverflow (or in this case, a CircleCI help thread). Worth keeping an eye on but probably not worth stressing.